core/src/main/java/io/github/fourlastor/game/demo/round/step/SimpleStep.kt

```diff
@@ -8,4 +15,17 @@ abstract class SimpleStep : Step<Unit>() {
     override fun enter(state: GameState, continuation: Consumer<Unit>, cancel: Runnable) {
         enter(state) { continuation.accept(Unit) }
     }
+
+    fun healthDeplete(stage: Stage, assetManager: AssetManager, x: Float, y: Float, damageAmount: Int): Action {
+        // (sheerst) - can we do this without having to pass stage and assetmanager?
+        return Actions.run {
+            val labelStyle = Label.LabelStyle(assetManager.get("fonts/quan-pixel-16.fnt"), Color.RED)
+            val healthDeplete = Label((-damageAmount).toString(), labelStyle)
+            healthDeplete.setPosition(x, y)
+            val duration = 0.5f
+            healthDeplete.addAction(Actions.moveTo(x, y + 18, duration, Interpolation.linear))
+            healthDeplete.addAction(Actions.fadeOut(duration, Interpolation.linear))
```

Comment:
might wanna check if this also removes the actor, or if you need to make this a sequence to remove the actor at the end